config/environments/development.rb

```diff
@@ -44,6 +44,18 @@
       secret_access_key: ENV["AWS_SECRET_KEY_ID"]
     }
   }
+  config.action_mailer.raise_delivery_errors = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+      address: "smtp.gmail.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.